src/main/scala/com/github/athieriot/CleanAfterExample.scala

```diff
@@ -1,16 +1,24 @@
 package com.github.athieriot
 
-import scala.collection.JavaConversions._
-import com.mongodb.{ MongoClient, ServerAddress }
-import org.specs2.specification.AfterExample
+import org.specs2.specification.AfterEach
+import reactivemongo.api.MongoDriver
 
-trait CleanAfterExample extends AfterExample {
+import scala.concurrent.Await
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+
+trait CleanAfterExample extends AfterEach {
   self: EmbedConnection =>
 
-  lazy val mongoClient = new MongoClient(new ServerAddress(network.getServerAddress(), network.getPort()));
+  def getConn = {
+    val driver = new MongoDriver
+    val db = driver.connection(List(s"127.0.0.1:${network.getPort}"))("testdb")
```

Comment:
ReactiveMongo gives an opportunity to run a custom command, so there is a way to get a list of all databases. I've made a [pull request](https://github.com/mtraina/specs2-embedmongo/pull/1) with an implementation of such a command.